readthedocs/donate/templates/donate/promo_detail.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Promo Detail" %}{% endblock %}
+
+{% block content %}
+
+<h1> Promo Results </h1>
+
+{% for promo in promos %}
+
+<h3>
+Results for {{ promo.name }} ({{ promo.analytics_id }}) over last {{ days }} days.
+</h3>
+
+<div class="example" style="width: 30%;">
+<a href="{{ promo.link }}"><img width=120 height=90 src="{{ promo.image }}"></a>
+<br>
+{{ promo.text }}
+
+</div>
+
+<br>
+
+<table>
+  <tr>
+    <th><strong>Day (UTC)</strong></th>
+    <th><strong>Views</strong></th> 
+    <th><strong>Clicks</strong></th>
+    <th><strong>CTR</strong></th>
```

Comment:
The `<strong>` here should also be replace be css rules. We shouldn't dictate styling in the HTML